### ISSUE TITLE
Is Docker volume create and use as external volume in docker-compose to persist data is supported

### ISSUE THREAD

**Issue by vairakkumar-vairaperumal** [louislam/uptime-kuma#3104]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Readme states: ' Please use a local volume only. Other types such as NFS are not supported."

Is external volume to persist data supported, this will allow to down uptime-kuma and bring it back up. If not what is the best method to persist data.

`
# Simple docker-compose.yml
# You can change your port or volume location

version: '3.3'

services:
  uptime-kuma:
    image: louislam/uptime-kuma:1
    container_name: uptime-kuma
    volumes:
      - uptime-kuma-data:/app/data
    ports:
      - 3001:3001  # <Host Port>:<Container Port>
    restart: always

volumes:
  uptime-kuma-data:
    external: true
    name: uptime-kuma-data

`

### 📝 Error Message(s) or Log

no error message

### 🐻 Uptime-Kuma Version

1.21.2

### 💻 Operating System and Arch

ubuntu 20.04

### 🌐 Browser

112.0.5615.137 (Official Build) (64-bit)

### 🐋 Docker Version

Docker version 20.10.14, build a224086

**Comment by louislam** [comment]
Yes, it is persistent. The message means that you should not use a network drive as the volume.
